examples/server.rs

```diff
@@ -40,7 +37,7 @@ async fn run_server(path: String) {
 	};
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Is one thread going to be enough?

Reply:
I don't think there's a reason here to use more than one thread - I tested examples locally on Linux and Windows and it's working for me.